cdk/src/functions/enable_controltower/index.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from awsapilib import ControlTower
+
+
+CREATE = 'Create'
+
+
+def handler(event, _):
+    RequestType = event["RequestType"]
+    control_tower_role_arn = os.getenv(
+        'AWSAPILIB_CONTROL_TOWER_ROLE_ARN'
+    )
+
+    tower = ControlTower(control_tower_role_arn)
+
+    logging_account_email = event['ResourceProperties'].get(
+        'LOG_ARCHIVE_AWS_ACCOUNT_EMAIL'
+    )
+    audit_account_email = event['ResourceProperties'].get(
+        'AUDIT_AWS_ACCOUNT_EMAIL'
+    )
+    if RequestType == CREATE:
+        tower.deploy(
+            logging_account_email=logging_account_email,
+            security_account_email=audit_account_email,
+            retries=50,
+            wait=5
+        )
+
+    return {}
```

Comment:
Does it make sense to log something for the end user in case some error occurs?

Reply:
If there was an error, the exception would be thrown and logged, anyways. We can add logging, though. I just didn't see any value in it. If we want real logging, then let's please log the entire stack trace, not just the message.